portal/views/user.py

```diff
@@ -2085,7 +2178,8 @@ def invite(user_id):
     return jsonify(message=message)
 
 
-@user_api.route('/user/<int:user_id>/messages')
+@user_api.route('/user/<int:user_id>/messages', methods=('OPTIONS','POST'))
```

Comment:
Did you intend to alter the method GET -> POST?  I understand the addition of OPTIONS, but this is why there's a failing test...

Reply:
I think that was already there, or I did that inadvertently. Nice catch (exclamation). I will fix.